### PR TITLE
MGDAPI-89: Update ConsoleLink reconciliation

### PR DESCRIPTION
**Pull request by sergioifg94**
# Description

Update reconciliation of ConsoleLink CRs to:

1. Add link to SSO as "API Management SSO"
2. Rename link to 3scale as "API Management"
3. Group these links into "OpenShift Managed Services" section

## Verification steps

In order to fully verify this PR:

1. Install the operator from master
2. Once the operator is installed, check that there's a 3scale link in the Applications menu
3. Change the operator to this PR
4. Wait for it to start and fully reconcile
5. Verify that the Applications menu contains:
    ```
    OpenShift Managed Services

    API Management
    API Management SSO
    ```
6. Uninstall the operator by deleting the RHMI CR
7. Verify that the links have been removed

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: pkg/products/rhsso/reconciler.go [hunk 1]
```diff
@@ -195,6 +202,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	if err := r.reconcileConsoleLink(ctx, serverClient); err != nil {
```
Comment (philbrookes) [inline pkg/products/rhsso/reconciler.go:205]: I think we should only be doing this on managed-api install type.

Comment (sergioifg94) [reply pkg/products/rhsso/reconciler.go:205]: Good point. Should we also be keeping the previous text for 3scale ("3scale API Management") when it's not managed-api?

Comment (philbrookes) [reply pkg/products/rhsso/reconciler.go:205]: I think calling it API Management in RHMI 2.x is fine.

Comment (philbrookes) [reply pkg/products/rhsso/reconciler.go:205]: I don't think it's used much in RHMI 2.x anyway, I'd say it's mostly accessed from the solution explorer.

Comment (sergioifg94) [reply pkg/products/rhsso/reconciler.go:205]: Cool, I just updated the PR with the condition to ensure the ConsoleLink is not created if the type isn't managed-api:
https://github.com/integr8ly/integreatly-operator/pull/1190/files#diff-74bcf7e72cc996549e6ed196e756ab37R410-R414

**Pull request merged**